dns/route53/main.tf

```diff
@@ -0,0 +1,27 @@
+variable "entries" {
```

Comment:
This module must define what version of Terraform and AWS provider is required to use it.

Reply:
Done.